tools/extensions/generate_extension_db.py

```diff
@@ -91,7 +102,7 @@ def GetExtensionMetadata(target):
   if NumRobustToDownstreamNetworkFilters(extension_db) != NumReadFiltersFuzzed():
     raise ExtensionDbError('Check that all network filters robust against untrusted'
                            'downstreams are fuzzed by adding them to filterNames() in'
-                           'test/extensions/filters/network/common/uber_per_readfilter.cc')
+                           '/source/test/extensions/filters/network/common/uber_per_readfilter.cc')
```

Comment:
Why the `/source/` prefix for the test dir? This is something that makes sense in the Docker environment but not to any developer working outside it.

Reply:
i think i added it because previously this path would be `./test/exten...` whereas now its inside the bazel build dir so i wasnt sure if it would be accessible - perhaps i need to just make sure that `test/extensions/...` is part of the build

Reply:
ok so i removed the path prefix from the error comment - the default would still be that path tho